.idea/hedera-network-e2e.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We shouldn't check in the .idea folder and add it to gitignore. Especially since this isn't a real project that needs anything more than a simple text editor.

Reply:
Removed .idea folder